PRS2001-3000/PRS2211Athanasi.xml

```diff
@@ -39,19 +39,21 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
          <change who="PL" when="2016-03-21"> Created file from
                                                   google spreadsheet </change>
          <change who="ES" when="2016-02-09">CREATED: person</change>
+         <change when="2024-07-12" who="MKr">Added relations and link to Wikidata</change>
       </revisionDesc>
    </teiHeader>
    <text xml:base="https://betamasaheft.eu/">
       <body>
          <listPerson>
-            <person sex="1">
-               <persName>
-                  
-                  Athanasius Gammālā
-               </persName>
-               <occupation> eccl. </occupation>
+            <person sex="1" sameAs="wd:Q2868987">
+               <persName xml:lang="en" xml:id="n1">Athanasius Gammālā</persName>
```

Comment:
I wonder, whether this is proper English, if it has diacritical signs (ā). Wikidata and Wikipedia calls him "Athanasius I Gammolo". Probably it is better to avoid a mixture of academic and non-academic title.
I propose:
<persName xml:lang="en" xml:id="n1">Athanasius I Gammolo</persName>